config/deploy.rb

```diff
@@ -0,0 +1,55 @@
+# config valid only for current version of Capistrano
+# bundle exec cap production deploy
+# bundle exec cap production rake task=db:seed
+lock '3.4.0'
+# set :use_sudo, true
+set :repo_url, 'https://github.com/pustovalov/app.git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.